kbs/src/plugins/sample.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2024 by Alibaba.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! This is a sample to implement a client plugin
+
+use actix_web::{http::Method, HttpResponse};
+use serde::Deserialize;
+
+use super::{plugin_manager::ClientPlugin, Result};
+
+#[derive(Deserialize, Clone, Debug, PartialEq)]
+pub struct SampleConfig {
+    pub item: String,
+}
+
+pub struct Sample {
+    _item: String,
+}
```

Comment:
Can you update the `kbs-client` to take requests for the Sample plugin?

Reply:
Yes we can! This would give a good implementation example